package.json

```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "test": "jest --verbose",
-    "pretest": "npm run lint",
+    "pretest": "npm run lint && require-self",
+    "prepare": "require-self",
```

Comment:
Need the npm install require-self here too or npm publish will fail. (Yeah it's not great but there is no other way due to the absence of a "local-install" npm script)

Reply:
done